src/loader.ts

```diff
@@ -26,34 +25,6 @@ module.exports = function (source: any) {
 
     let codeTransformed = source
 
-    if (resourcePath.endsWith('.ts')) {
```

Comment:
It would be great to have an example in the README.md explaining how this Webpack plugin could be used with TypeScript.

Reply:
I started looking into this, but it's quite complicated. I was not able to get a working setup.

To be fair, we also need good docs for how to integrate TypeScript with the Rollup plugin as well. I suspect it's equally challenging.

I'm going to file an internal ticket to do this.